tests/helpers.py

```diff
@@ -479,6 +479,26 @@ def __init__(self, **kwargs) -> None:
         self.invoked_from_error_handler = kwargs.get('invoked_from_error_handler', False)
 
 
+class MockInteraction(CustomMockMixin, unittest.mock.MagicMock):
+    """
+    A MagicMock subclass to mock Interaction objects.
+
+    Instances of this class will follow the specifications of `discord.Interaction`
+    instances. For more information, see the `MockGuild` docstring.
+    """
+    # spec_set = context_instance
```

Comment:
Could try and make an instance for `Interaction` like was done with context, although if the tests work as is it's probably fine to just remove this. I can't remember exactly what it does :P

Reply:
The tests work fine, so I will just remove the comment